### PR TITLE
Less slick update of relative to absolute imports

### PR DESCRIPTION
**Pull request by cmurray-astro**
I went through and updated the relative beast imports to absolute imports. I did not notice that @eteq was already working on this, and I ran into many unforced errors and did a lot of brute-force fixing... but as far as I can tell I think it's working!

### CODE REVIEW

**Comment by eteq** [comment]
Note: we should close #391 once this is in.

File: beast/observationmodel/noisemodel/toothpick.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
 
 from tqdm import tqdm
 
-from .noisemodel import NoiseModel
-from ..vega import Vega
+from beast.observationmodel.noisemodel.noisemodel import NoiseModel
```
Comment (eteq) [inline beast/observationmodel/noisemodel/toothpick.py]: This one is painful to look at (it's right, but I wonder if it's a sign that the `from noisemodel import *` trick should be used in `beast/observanmodel/noisemodel/__init__.py`? (cc @karllark)

(not necessarily for this PR, but as a broader task)

**Comment by cmurray-astro** [comment]
Thanks, Erik! I'll fix the first one right away, will let @karllark weigh in on the second.

File: beast/observationmodel/noisemodel/toothpick.py (hunk 1 above)
Comment (karllark) [reply beast/observationmodel/noisemodel/toothpick.py]: Using __init__.py is make this prettier is something to think about.  I agree, not for this PR.

**Comment by karllark** [comment]
Closes #388.

**Pull request merged**